Components/index.html

```diff
@@ -7,10 +7,56 @@
     <link rel="stylesheet" href="main.css">
 </head>
 <body>
-    <div class="lighmode">
+    <div class="lightmode">
         <div class="rootDiv">
             <h1>CRUX UI</h1>
         </div>
+        <div>
+          <div>
+            <button class="button button-primary">Primary</button>
+            <button class="button button-secondary">secondary</button>
+            <button class="button button-success">success</button>
+            <button class="button button-warning">warning</button>
+            <button class="button button-info">info</button>
+            <button class="button button-error">error</button>
+            <button class="button button-monochrome">monochrome</button>
+          </div>
+          <br>
+          <div>
+            <button class="button button-primary-outlined">Primary</button>
+            <button class="button button-secondary-outlined">secondary</button>
+            <button class="button button-success-outlined">success</button>
+            <button class="button button-warning-outlined">warning</button>
+            <button class="button button-info-outlined">info</button>
+            <button class="button button-error-outlined">error</button>
+            <button class="button button-monochrome-outlined">monochrome</button>
+          </div>
+          <br>
+          <div>
+            <button class="button button-secondary button-lg">secondary</button>
+            <button class="button button-success">success</button>
+            <button class="button button-warning button-sm">warning</button>
+          </div>
+          <br>
+          <div>
+
+            <button class="button button-primary" disabled>Primary</button>
+            <button class="button button-secondary" disabled>secondary</button>
+            <button class="button button-success" disabled>success</button>
+            <button class="button button-warning" disabled>warning</button>
+            <button class="button button-info" disabled>info</button>
+            <button class="button button-error" disabled>error</button>
+            <button class="button button-light" disabled>light</button>
+            <button class="button button-monochrome" disabled>monochrome</button>
+            <button class="button button-primary-outlined" disabled>Primary</button>
+            <button class="button button-secondary-outlined" disabled>secondary</button>
```

Comment:
If disabled buttons has no-drop cursor, it'd look good.

Reply:
sure